exercises/secret-handshake/instructions.md

```diff
@@ -18,7 +17,7 @@ The actions for each number place are:
 Let's use the number `9` as an example:
 
 - 9 in binary is `1001`.
-- The digit that is farthest to the right is 1, so the first action is `wink`.
+- The digit that is closest to the right is 1, so the first action is `wink`.
```

Comment:
Maybe something like:

> - The rightmost digit is 1, so the first action is `wink`.

Reply:
As mentioned in the issue, I deliberately changed the wording to provide two different explanations of the same point (rightmost). There shouldn't be a comprehension issue in something this minor, but anyway.

Reply:
I agree. "farthest to the right" is pretty clear. I'm not entirely sure where the comprehension issue is coming from.

I'm not a fan of this change as it is written, per my comment, so I'm not going to approve as is. But if others think this wording is better, I'll leave it to them :)

Reply:
I would like to understand what the comprehension issue is.

Earlier in the discussion the explanation for this change was that some tracks might have numbers larger than 31 in the test data.

Reply:
Ok, so a few things.

We've used `right-most` earlier already. Using `farthest to the right` was a way to vary the wording, in case that helps comprehension for some reason.

_farthest to the right_ is more correct, to my mind, than _closest to the right_, because you have to be closest to a _something_, and _the right_ isn't a something. _farthest to the right_ is a fixed expression that means right-most.

Reply:
> We've used right-most earlier already. Using farthest to the right was a way to vary the wording, in case that helps comprehension for some reason.
That's what I thought, too.

> farthest to the right is more correct, to my mind
I agree with you, I'll change this.

Reply:
https://www.thefreedictionary.com/Rightmost

> Rightmost: Farthest to the right

Yes. "Farthest to the right" is correct. That's been pointed out several times, both here and on the forum.

Reply:
Oh. So does this PR have any use, then? We can close it if not.

Reply:
I don't think this PR has much use, either. Most the comments on the forum when this was raised were along the lines of, "I don't see anything wrong". I'm in favor of closing this out.

Reply:
Yeah, I think we can close it.

Thanks for taking the time to think through this.

In the future I would suggest that you work out the details in the forum befor opening the issue.